test/NuGet.Services.Staging.EndToEndTests/StagingClient/NuGetLoggerAdapter.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Xunit.Abstractions;
+
+namespace NuGet.Services.Staging.EndToEndTests
+{
+    public class NuGetLoggerAdapter : NuGet.Common.ILogger

```

Comment:
fixed

---
In reply to: [64173941](https://github.com/NuGet/NuGet.Staging/pull/49#discussion_r64173941) [](ancestors = 64173941)